aws-lc-rs/src/signature.rs

```diff
@@ -697,6 +773,22 @@ pub static ECDSA_P384_SHA384_ASN1_SIGNING: EcdsaSigningAlgorithm =
 pub static ECDSA_P384_SHA3_384_ASN1_SIGNING: EcdsaSigningAlgorithm =
     EcdsaSigningAlgorithm(&ECDSA_P384_SHA3_384_ASN1);
 
+/// Signing of ASN.1 DER-encoded ECDSA signatures using the P-521 curve and SHA-224.
+pub static ECDSA_P521_SHA1_ASN1_SIGNING: EcdsaSigningAlgorithm =
+    EcdsaSigningAlgorithm(&ECDSA_P521_SHA1_ASN1);
+
```

Comment:
* I suspect the addition of `ECDSA_P521_SHA1_ASN1_SIGNING` was unintentional. (?) I think support for verification w/ P521+SHA1, but not signing, is the right approach.

Reply:
Ah yeah I originally supported signing with sha1, and then thought I had removed all of the references. Looks like I only removed the fixed signing option.